File: Awesome.AI.Source/Awesome.AI/Access/Access.cs
using Awesome.AI.Common;
using static Awesome.AI.Variables.Enums;

namespace Awesome.AI.Access
{
    internal class AccessApi
    {        
    }

    public class ApiRequestSetup
    {
        public List<MINDS> bots { get; set; }
    }

    public class ApiRequestUpdate
    {
        public MINDS mindtype { get; set; }
        public CASE _case { get; set; }
        public VALUE _value { get; set; }
    }

    public class ApiRequestAnswer
    {
        public MINDS mindtype { get; set; }
    }

    public class ApiInstance
    {
        public string o_json {  get; set; }
        public string s_hits_json { get; set; }
        public string s_units_json { get; set; }
        public MyOutVars o_vars { get; set; }
        public MyStats s_vars { get; set; }
        public MINDS mindtype { get; set; }
    }
}

File: Awesome.AI.Python/Access/Access.py
from dataclasses import dataclass, field
from typing import Any
from awesome_ai.Variables.Enums import CASE, MINDS, VALUE


class AccessApi:
    pass


@dataclass
class ApiRequestSetup:
    bots: list[MINDS] = field(default_factory=list)


@dataclass
class ApiRequestUpdate:
    mindtype: MINDS = MINDS.BASIC
    _case: CASE = CASE.NONE
    _value: VALUE = VALUE.EMPTY


@dataclass
class ApiRequestAnswer:
    mindtype: MINDS = MINDS.BASIC


@dataclass
class ApiInstance:
    o_json: str = ""
    s_hits_json: str = ""
    s_units_json: str = ""
    o_vars: Any = None
    s_vars: Any = None
    mindtype: MINDS = MINDS.BASIC

